ToDoApp/ToDoApp/SignUp/SignUpView.swift

```diff
@@ -0,0 +1,124 @@
+                    .foregroundColor(Color(.secondaryText))
+                Rectangle()
+                    .foregroundColor(.white)
+                    .frame(maxHeight: 62)
+                VStack(alignment: .leading) {
+                    TextField("Name", text: $viewModel.name)
+                        .padding(.horizontal, 16.0)
+                        .textContentType(.name)
+                        .frame(minHeight: 60)
+                        .introspectTextField(customize: {
+                            $0.clearButtonMode = .whileEditing
+                        })
+                    Divider()
+                    TextField("Email", text: $viewModel.email)
+                        .padding(.horizontal, 16.0)
+                        .keyboardType(.emailAddress)
+                        .textContentType(.emailAddress)
+                        .frame(minHeight: 60)
+                        .introspectTextField(customize: {
+                            $0.clearButtonMode = .whileEditing
+                        })
+                    Divider()
+                    SecureField("Password", text: $viewModel.password)
+                        .padding(.horizontal, 16.0)
+                        .textInputAutocapitalization(.never)
+                        .frame(minHeight: 60)
+                        .introspectTextField(customize: {
+                            $0.clearButtonMode = .whileEditing
+                        })
+                    Divider()
+                    HStack {
+                        Button(action: {
+                            viewModel.acceptTermsAndConditionsCheckBoxTapped()
+                        }, label: {
+                            Image(systemName: (viewModel.isAcceptTermsAndConditionsChecked ?  "checkmark.square.fill" : "squareshape"))
+                                .frame(minWidth: Constants.checkBoxSize, minHeight: Constants.checkBoxSize)
+                        }
+                        )
+                        Text("I agree with our Terms and Conditions")
+                        { string in
+                            string.foregroundColor = Color(.secondaryText)
+                            string.font = .caption
+                            if let range = string.range(of: "Terms") {
+                                string[range].foregroundColor = Color(.checkmarkButton)
+                            }
+                            if let range = string.range(of: "Conditions") {
+                                string[range].foregroundColor = Color(.checkmarkButton)
+                            }
+                        }
+                    }
+                    .padding(16)
+                    NavigationLink("", destination:  ToDoListView(viewModel: ToDoListViewModel(taskService: ToDoServiceImpl(dateChecker: Calendar.current, repo: Database.database().reference() , userId: viewModel.userId))), isActive: $viewModel.moveToListView)
+                }
```

Comment:
I think it's not a correct handle for navigating to list view screen, but if you have any recommendation for me, please let me know so that I can fix it.